frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -228,7 +256,13 @@ class Multiselect extends React.PureComponent<Props, State> {
             value={this.valueFromState}
             disabled={disabled}
             size={"compact"}
+            noResultsMsg={
+              this.props.element.maxSelections !== 0
+                ? this.getNoResultsMsg()
+                : "No results"
+            }
             filterOptions={this.filterOptions}
+            closeOnSelect={false}
```

Comment:
Does this end up being a behavioral change? We'd want to check with product if that's the case

Reply:
Yeah product actually wanted this.